BlueDBOnDisk/src/main/java/io/bluedb/disk/collection/BlueCollectionOnDisk.java

```diff
@@ -47,16 +47,16 @@
 	private final RollupScheduler rollupScheduler;
 	private final CollectionMetaData metaData;
 
-	public BlueCollectionOnDisk(BlueDbOnDisk db, String name, Class<T> valueType, Class<? extends BlueKey> keyType2) throws BlueDbException {
+	public BlueCollectionOnDisk(BlueDbOnDisk db, String name, Class<T> valueType, Class<? extends BlueKey> keyType) throws BlueDbException {

```

Comment:
I would consider putting the key first. Usually key comes before value. Also, it would make sense to allow them to pass in extra classes to register after the value. Then the value and classes to register will be next to each other in the parameters list.

Reply:
Also, I'm wondering if it would make more sense to have a declareCollection method and a getCollection method. The former specifies the name, keyType, valueType, and classes to register. The latter just specifies the name. I know they can create a method so they only have to get the collection once, but it seems a little weird to pass in so many parameters every time they get the collection.

Reply:
I think these are good ideas and implemented both.